uni/lib/view/home/widgets/tracking_banner.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:flutter/material.dart';
+import 'package:uni/generated/l10n.dart';
+
+class BannerWidget extends StatelessWidget {
```

Comment:
Can this widget have a more explicit name?
If I was looking into code without running the app, I knew this was a banner but for what purpose?

Changing BannerWidget to TrackingBanner is an idea.